### PR TITLE
git -> 2.29.2

### PR DESCRIPTION
**Pull request by satmandu**
bring to latest patch level

Works properly & w/ install.sh:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/git.rb [hunk 1]
```diff
@@ -3,32 +3,36 @@
...
     system 'autoreconf -i'
-    system "./configure --with-openssl=#{CREW_PREFIX}/etc/ssl --without-tcltk #{CREW_OPTIONS} --with-perl=#{CREW_PREFIX}/bin/perl --with-python=#{CREW_PREFIX}/bin/python3 \
---with-gitconfig=#{CREW_PREFIX}/etc/gitconfig --with-gitattributes=#{CREW_PREFIX}/etc/gitattributes"
+    system "./configure \
+    --with-openssl=#{CREW_PREFIX}/etc/ssl \
+    --without-tcltk #{CREW_OPTIONS} \
+    --with-perl=#{CREW_PREFIX}/bin/perl \
+    --with-python=#{CREW_PREFIX}/bin/python3 \
+    --with-gitconfig=#{CREW_PREFIX}/etc/gitconfig \
+    --with-gitattributes=#{CREW_PREFIX}/etc/gitattributes"
```
Comment (uberhacker) [inline packages/git.rb:35]: We'll need install.sh updated also.  Please test a fresh install on all architectures.

Comment (satmandu) [reply packages/git.rb:35]: Have tested this in a armv7l vm. Will also do so on i686 & x86_64 vms.

Comment (satmandu) [reply packages/git.rb:35]: works fine on x86_64

Comment (satmandu) [reply packages/git.rb:35]: works fine on i686 too.

**Comment by satmandu** [comment]
This should be good to go.

**Pull request merged**